source/blog/2018-11-19-ember-3-6-released.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Ember 3.6 Released
+author: Melanie Sumner, Kenneth Larsen, Chris Garrett
+tags: Releases, 2018, 3, 3.6
+responsive: true
+---
+
+Today the Ember project is releasing version 3.6 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.7 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

Comment:
```suggestion
Today the Ember project is releasing version 3.6 of Ember.js, Ember Data, and Ember CLI. Highlights include public API support for ES6 classes and two new RouterService methods.

This release kicks off the 3.7 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

I think we should add a TLDR for newsletters to grab and use. I did this for the last release blog post after lots of back and forth about restructuring the format of these posts, and the summary compromise was to add a short blurb at the top and not change structure. This is an optional addition but I think it helps marketing and visibility.